packages/evolu-common/src/Evolu.ts

```diff
@@ -731,16 +731,16 @@ const EvoluCommon = Layer.effect(
       makeStore<SyncState>({ _tag: "SyncStateInitial" }),
     );
     const mutate = yield* _(Mutate);
-    const ownerStore = yield* _(makeStore<Owner | null>(null));
+    const ownerStore = yield* _(makeStore<O.Option<Owner>>(O.none()));
     const loadingPromises = yield* _(LoadingPromises);
     const appState = yield* _(AppState);
 
     dbWorker.onMessage = (output): void =>
       Match.value(output).pipe(
         Match.tagsExhaustive({
-          onError: ({ error }) => errorStore.setState(error),
+          onError: ({ error }) => errorStore.setState(O.some(error)),
           onQuery,
-          onOwner: ({ owner }) => ownerStore.setState(owner),
+          onOwner: ({ owner }) => ownerStore.setState(O.some(owner)),
```

Comment:
@steida In case `onOwner` callback expects as parameter  `owner: Owner|null` , then change this line to:

```ts
onOwner: ({ owner }) => ownerStore.setState(O.fromNullable(owner)),
```